Uni_life_tracker/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I was just wondering what this is being used for

Reply:
It showed automatically because there are more than two activities after main

Reply:
I think you should just delete them then because we arent using it

Reply:
Last time I deleted it caused an error for some reason

Reply:
I'll try again

Reply:
Just deleted them